packages/alfa-aria/src/features/th.ts

```diff
@@ -0,0 +1,11 @@
+import { Feature, AnyRole } from "../types";
+import * as Roles from "../roles";
+
+/**
+ * @see https://www.w3.org/TR/html-aria/#th
+ */
+export const Th: Feature = {
+  element: "th",
+  role: Roles.ColumnHeader || Roles.RowHeader,
```

Comment:
`||` is a boolean operator so the above would mean: `ColumnHeader` if truthy, otherwise `RowHeader`. Since `ColumnHeader` is always truthy, it will always be the result of the given expression. What we want in this case is to determine if the header if associated with a column or a row, which will dictate the implied role. Let's put our heads together and take a stab at it.